config/application.rb

```diff
@@ -273,6 +273,8 @@ class Application < Rails::Application
         child.helper helpers
       end
 
+      Facets.register(HostFacets::ReportedDataFacet, :reported_data)
```

Comment:
Note, that you need to explicitly set `dependant` to `:destroy`, via `set_dependant :destroy` I believe.